src/main/java/Chess/domain/ChessBoard.java

```diff
@@ -0,0 +1,95 @@
+package Chess.domain;
+
+import Chess.domain.ChessUnit.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class ChessBoard {
+    public static final int CHESSBOARD_ROW = 8;
+    public static final int CHESSBOARD_COLUMN = 8;
+    public static final int WHITE_SPECIAL_CHESS_UNIT_ROW = ChessBoard.CHESSBOARD_ROW - 1;
+    public static final int WHITE_PAWN_CHESS_INIT_ROW = ChessBoard.CHESSBOARD_ROW - 2;
+    public static final int BLACK_PAWN_CHESS_UNIT_ROW = 1;
+    public static final int BLACK_SPECIAL_CHESS_UNIT_ROW = 0;
+    public static final List<Class<? extends ChessUnit>> INITIAL_SPECIAL_CHESS_UNIT_POSITION = Arrays.asList(
+            Rook.class,
+            Knight.class,
+            Bishop.class,
+            Queen.class,
+            King.class,
+            Bishop.class,
+            Knight.class,
+            Rook.class
+    );
+
+    private final static ChessBoard instance = new ChessBoard();
+
+    public static ChessBoard getInstance() {
+        return instance;
+    }
+
+    private ChessUnit[][] chessBoard = new ChessUnit[CHESSBOARD_ROW][CHESSBOARD_COLUMN];
+
+    private ChessBoard() {
+        initializeChessGame();
+    }
+
+    public static int convertInputRowToDataRow(int row) {
+        return CHESSBOARD_ROW - row;
+    }
+
+    public void initializeChessGame() {
+        try {
+            initializeChessboard();
+            initializeChessUnit();
+        } catch (Exception e) {
+            initializeChessboard();
+        }
+    }
+
+    private void initializeChessboard() {
+        for (int i = 0; i < CHESSBOARD_ROW; i++) {
+            for (int j = 0; j < CHESSBOARD_COLUMN; j++) {
+                chessBoard[i][j] = new EmptyCell();
+            }
+        }
+    }
+
+    private void initializeChessUnit() throws Exception {
+        for (int i = 0; i < CHESSBOARD_COLUMN; i++) {
+            setUnitFromCell(BLACK_SPECIAL_CHESS_UNIT_ROW, i, INITIAL_SPECIAL_CHESS_UNIT_POSITION.get(i).getConstructor(ChessUnitColor.class).newInstance(ChessUnitColor.BLACK));
```

Comment:
한 줄에 점(.) 이 여러 개 들어가지 않게 합시다.
```suggestion
            setUnitFromCell(
                BLACK_SPECIAL_CHESS_UNIT_ROW, i, INITIAL_SPECIAL_CHESS_UNIT_POSITION
                .get(i)
                .getConstructor(ChessUnitColor.class)
                .newInstance(ChessUnitColor.BLACK)
            );
```
이렇게 하면 가독성에도 도움이 되고 디버깅 시에도 한 줄씩 메모리상의 값을 검사해볼 수 있습니다.